app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,29 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#">Brand</a>
+    </div>
+
+    <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav navbar-right">
+        <% if user_signed_in? %>
+          <li><a href="#" class="navbar-link no-pointer">Welcome, <%= current_user.name %></a></li>
```

Comment:
I would like to see the name displayed in title size.  I was trying to look at how to do it before the name gets saved to the db.  I am guessing it has to be done in the controller.  With Devise, I saw a way to generate [controllers](https://github.com/plataformatec/devise/wiki/Tool:-Generate-and-customize-controllers) to override the standard ones.  This seemed pretty complex and I want to know if that is the best way before I dive in head first.